### PR TITLE
Faq Page (intro section + questins) 

### PR DESCRIPTION
**Pull request by abdulbasit-web**
Abdulbasit web/issue5
this branch is issue5 in the project 
it has FaqPage introsection and question section

### CODE REVIEW

File: src/components/FaqIntro/FaqIntro.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            </h1>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit.
+              Perspiciatis officiis recusandae voluptatibus! Natus voluptatum
+              esse temporibus saepe quo omnis alias!
+            </p>
+          </div>
+        </Col>
+        <Col lg="6" className="">
+          <img src={intro} className="img-fluid IntroSection__image" alt="" />
```
Comment (MustafaNawzat) [inline src/components/FaqIntro/FaqIntro.js:22]: change `IntroSection__image` to `introSectionImage`

we are using camelCase for classes naming.

File: src/components/Questions/QuestionItem.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  QuestionItem.propTypes = {
+    question: PropTypes.object.isRequired,
+    text: PropTypes.string.isRequired,
+    answer: PropTypes.string.isRequired,
+    id: PropTypes.number.isRequired,
+  };
+
+  return (
+    <Accordion>
+      <Card className="qustion__card my-4">
```
Comment (MustafaNawzat) [inline src/components/Questions/QuestionItem.js:20]: use camelCase for classes naming

File: src/components/Questions/Questions.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+.card-header p {
+  margin: 0;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 24px;
+  line-height: 28px;
+  /* identical to box height */
```
Comment (MustafaNawzat) [inline src/components/Questions/Questions.css:25]: remove unused code before making any Pull Request

File: src/components/Questions/Questions.css [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+  letter-spacing: -0.5px;
+
```
Comment (MustafaNawzat) [inline src/components/Questions/Questions.css:28]: remove empty lines inside code block.

File: src/components/Questions/__test__/Qustions.test.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Qustions from '../Qustions';
+import renderer from 'react-test-renderer';
+// import { render } from '@testing-library/react';w
+
+// test('Qustions is rendering ', () => {
```
Comment (MustafaNawzat) [inline src/components/Questions/__test__/Qustions.test.js:6]: remove commented code

File: src/containers/FaqPage/FaqPage.jsx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import FaqIntro from '../../components/FaqIntro/FaqIntro';
+import Qustion from '../../components/Questions/Qustions';
```
Comment (MustafaNawzat) [inline src/containers/FaqPage/FaqPage.jsx:3]: fix this typo Qustion >> Question

File: src/components/IntroSection/IntroSection.js [hunk 7]
```diff
@@ -9,7 +9,7 @@ function IntroSection() {
     <div className="introSection">
       <img src={penIntro} className="penIntro" alt="" />
```
Comment (MustafaNawzat) [inline src/components/IntroSection/IntroSection.js:10]: why IntroSection.js file is here? don't make changes in files that are not related to this branch.

undo this file changes from this branch.

File: src/components/Questions/Qustions.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import { Container } from 'react-bootstrap';
+import QuestionItem from './QuestionItem';
+
+// const decoratedOnClick = useAccordionToggle(eventKey, onClick);
```
Comment (MustafaNawzat) [inline src/components/Questions/Qustions.js:5]: delete unused lines of code.

File: src/components/Questions/Questions.css [hunk 9]
```diff
@@ -3,7 +3,7 @@
   margin: 0;
```
Comment (MustafaNawzat) [inline src/components/Questions/Questions.css:3]: styles should be in scss file and not css

Questions.css >> Questions.scss

File: src/components/Questions/__test__/Qustions.test.js [hunk 10]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```
Comment (MustafaNawzat) [inline src/components/Questions/__test__/Qustions.test.js:1]: move the test to tests file in the src

File: src/App.js [hunk 11]
```diff
@@ -2,14 +2,12 @@ import React from 'react';
...
 import HomePage from './containers/HomePage/HomePage';
-import Footer from './containers/Footer/Footer';
 
 function App() {
   return (
     <div>
       <Nav />
-       <HomePage />
-      <Footer />
+      <HomePage />
```
[the remark this answers is not in the document]
Comment (abdulbasit-web) [reply src/App.js:10]: because in footer folder it not contain Footer.js
even after git pull

**Comment by MustafaNawzat** [comment]
@abdulbasit-web please fix yarn.lock conflict so I can merge
